src/GitVersion.Core/Model/GitVersionContext.cs

```diff
@@ -9,9 +8,9 @@ namespace GitVersion;
 public class GitVersionContext
```

Comment:
Feel free to move every other file inside the `Model` directory, such as `GitVErsionContext.cs`, as well. It would be good to get rid of the `GitVersion.Model` namespace entirely.